WooCommerce/WooCommerceUITests/UITests.xctestplan

```diff
@@ -9,7 +9,7 @@
     }
   ],
   "defaultOptions" : {
-
+    "testRepetitionMode" : "retryOnFailure"
```

Comment:
How many times will this retry? I couldn't easily find docs on it.

I think this'll be nice to have for sure 🙂

Reply:
With retry on failure, it will retry to up to 3 test runs, not configurable in Xcode 13.2.

<img width="449" alt="image" src="https://user-images.githubusercontent.com/1218433/153559239-623ed5b5-43f2-4e47-81d2-e4ee808f4d65.png">